.github/workflows/ci.yml

```diff
@@ -24,7 +24,19 @@ jobs:
         run: docker compose up --wait minio db api-legacy api dataextraction
 
       - name: Run dotnet tests
-        run: dotnet test BDMS.sln -c Release --no-build --verbosity normal --filter TestCategory!=LongRunning
+        run: dotnet test BDMS.sln -c Release --no-build --verbosity normal --filter TestCategory!=LongRunning --logger trx
+
+      - name: Upload test results
+        uses: actions/upload-artifact@v4
+        with:
+          name: dotnet-test-results
+          path: ${{ github.workspace }}/**/TestResults/**/*
+
+      - name: Publish test results
+        uses: EnricoMi/publish-unit-test-result-action@v2
+        if: always()
```

Comment:
Q: [always()](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#always) wird auch ausgeführt, wenn abgebrochen wurde. ist das so gewollt?